financial-templates-lib/ExpiringMultiPartyClient.js

```diff
@@ -121,6 +121,12 @@ class ExpiringMultiPartyClient {
     for (const address of this.sponsorAddresses) {
       const liquidations = await this.emp.methods.getLiquidations(address).call();
       for (const [id, liquidation] of liquidations.entries()) {
+        // Liquidations that have had their rewards withdrawn will still show up here but have their "sponsor" property deleted.
+        // We can skip them.
+        if (liquidation.sponsor === "0x0000000000000000000000000000000000000000") {
```

Comment:
@ptare added this here because I realize that after liquidations are deleted (via `withdrawLiquidation`), they still appear in `getLiquidations()` but all their properties are zeroed out.

Reply:
Will this work if the sponsor has withdrawn, but the liquidator hasn't, for instance?

Reply:
No, good catch. I'm thinking I can check `if(liquidation.state === "0")` here instead, wdyt?